src/drivers/mct/cime_config/config_component.xml

```diff
@@ -473,6 +473,37 @@
     </desc>
   </entry>
 
+  <entry id="RESUBMIT_AT_START">
```

Comment:
If DOUT_S is TRUE than the new job should depend on the completion of case.st_archive, not on case.run.